src/components/play-bar/desktop/PlayBar.js

```diff
@@ -396,38 +398,49 @@ class PlayBar extends Component {
           </div>
 
           <div className={styles.optionsRight}>
+            <VolumeBar
+              defaultValue={100}
+              granularity={VOLUME_GRANULARITY}
+              onChange={this.updateVolume}
+            />
+            <div className={styles.toggleRepostContainer}>
+              <Tooltip
+                text={repostText}
+                disabled={favoriteAndRepostIsDisabled}
+                mount='parent'
+                placement='top'
+              >
+                <span>
```

Comment:
im guessing this <span> element is here because the tooltip needs something that can accept a ref to know how to anchor. if youre interested, we can update the repost/favorite buttons to accept a ref with `React.forwardRef`

Reply:
ahh i see! so it's injecting props, nice find. so yeah keeping span is prob fine, but this probably also shows that we should be passing all "button" props through too, so maybe an action item in general in addition to forwardRef is allowing all button props to go through to button element. def a bit overkill in terms of eng effort though